sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -131,7 +130,7 @@ export interface ListRevisionsPage extends HttpResponseField<SyncTokenHeaderFiel
 
 // @public
 export interface ListSettingsOptions extends OptionalFields {
-    acceptDatetime?: string;
+    acceptDatetime?: Date;
```

Comment:
Should these be `acceptDateTime` with an uppercase 'T'?

Reply:
It's actually fronting for this header (Accept-Datetime) which leaves the 't' lowercase as well.

Reply:
I think this casing is confusing, personally. You remove the hyphen-minus from the header name and lower-case `Accept`, so might as well normalize the casing as well?

Reply:
Two votes to change the casing wins. :)

Reply:
Fixed :)